src/app/directive-way/directive-way.component.ts

```diff
@@ -17,6 +17,7 @@ export class DirectiveWayComponent {
     endpoint: `${environment.api}/files?uploadType=uploadx`,
     token: this.authService.getAccessToken(),
     maxChunkSize: 1024 * 1024 * 8,
+    storeIncompleteUploadUrl: false,
```

Comment:
The docs say that the default is `true`, but it seems it's `false`.

Reply:
That's a typo. Thanks.